lib/internal/worker/io.js

```diff
@@ -447,13 +455,21 @@ ObjectDefineProperties(BroadcastChannel.prototype, {
 defineEventHandler(BroadcastChannel.prototype, 'message');
 defineEventHandler(BroadcastChannel.prototype, 'messageerror');
 
+function markAsUncloneable(obj) {
+  if ((typeof obj !== 'object' && typeof obj !== 'function') || obj === null) {
+    return;
+  }
```

Comment:
Why is it a no-op, rather than throwing an error?

Reply:
I kept it to be consistent with `markAsUntransferable`. It's easier to use without throwing error, I think.

Also the doc is kept consistent with `markAsUntransferable`, to answer your other comments.